backend/src/routes/api/configmaps/index.ts

```diff
@@ -0,0 +1,54 @@
+import { KubeFastifyInstance } from '../../../types';
+import { FastifyRequest } from 'fastify';
+import { V1ConfigMap } from '@kubernetes/client-node';
+
+module.exports = async (fastify: KubeFastifyInstance) => {
+  fastify.get('/:name', async (request: FastifyRequest) => {
+    const params = request.params as {
+      name: string;
+    };
```

Comment:
We should avoid `as` as much as we can; it forces the type instead of defining it normally (which limits TS ability to help us)... `FastifyRequest` is a generic, so we can do this:

```
FastifyRequest<{ Params: { name: string } }>
```
And we'll get request.params typed.